game/audio.cc

```diff
@@ -423,7 +423,12 @@ struct Audio::Impl {
 	std::string selectedBackend = Audio::backendConfig().getValue();
 	Impl() {
 		populateBackends(portaudio::AudioBackends().getBackends());
-		std::clog << portaudio::AudioBackends().dump() << std::flush; // Dump PortAudio backends and devices to log.
+		{
```

Comment:
Why the extra braces?

Reply:
To limit the lifetime of logmsg to that block.